articles/desktop-flows/virtual-desktops.md

```diff
@@ -14,117 +14,97 @@ search.audienceType:
+With the Power Automate agent for virtual desktops, you can automate processes on virtual desktops just as easily as you can on physical devices. If your virtual desktop uses Citrix or Microsoft Remote Desktop Protocol (RDP), you can [capture UI elements](ui-elements.md), [deploy UI automation actions](actions-reference/uiautomation.md), and [create desktop flows using the recorder](recording-flow.md), just like on your physical desktop.
 
 > [!IMPORTANT]
-> - This is a preview feature.
-> - [!INCLUDE[cc_preview_features_definition](../includes/cc-preview-features-definition.md)]
-> - Automating on virtual desktops doesn't currently support the automation of webpages using browser automation actions. To automate webpages on virtual desktops, you can [handle them as desktop UI elements](desktop-automation.md), [use mouse, keyboard, and OCR actions](how-to/automate-using-mouse-keyboard-ocr.md), or [automate using images](images.md).
-
-
-To enable this functionality, install the **Power Automate agent for virtual desktops** on your virtual desktop. The agent communicates with the Power Automate for desktop application and facilitates the required interaction with it. The same agent can work with both Citrix and RDP virtual desktops.
+> This is a preview feature. [!INCLUDE[cc_preview_features_definition](../includes/cc-preview-features-definition.md)]
 
-Communication is performed through the same channel that the Citrix/RDP virtual desktop uses to connect to your machine. Therefore, a running agent is necessary for desktop flows to interact with a virtual desktop, and the agent should be up and running for both design and run time.
-
-The agent starts running every time the user logs in to the virtual desktop, as the launching of the agent is set as a scheduled task. If the agent isn't running on your virtual desktop, the connection with Power Automate for desktop can't be established. In this case, relaunch the agent manually.  
+The Power Automate agent for virtual desktops must be running both while you're designing a flow and when the flow runs. The agent starts automatically when a user logs in to the virtual desktop. If it isn't running on your virtual desktop, launch the agent manually.  
 
 ## Prerequisites
 
-* Windows 10 Pro, Windows 10 Enterprise, Windows 11 Pro, Windows 11 Enterprise, Windows Server 2016, Windows Server 2019 or Windows Server 2022
-
-*	Power Automate for desktop should be installed on the user’s host machine. Users must download the installer from the Power Automate portal and install it manually.
-
-*	Power Automate agent for virtual desktops should be installed on the Citrix/RDP virtual desktop. The agent installation requires admin rights.
+- Your physical device and virtual machine must be running Windows 10 Pro, Windows 10 Enterprise, Windows 11 Pro, Windows 11 Enterprise, Windows Server 2016, Windows Server 2019, or Windows Server 2022.
```

Comment:
@rhanajoy It's important to mention that the users have to download the installer manually through the portal, for example not MS store. Also, we have to mention the admin rights specification.

Reply:
Perfect, we can do that